tests/fileTypeTests.py

```diff
@@ -123,6 +124,13 @@ def test_award_fin_mixed(self):
         jobId = self.jobIdDict["awardFinMixed"]
         self.passed = self.run_test(
             jobId, 200, "finished", 17206, 5, "complete", 87, True)
+        # Test that whitespace is converted to null
+        rowThree = self.interfaces.validationDb.session.query(AwardFinancial).filter(AwardFinancial.row_number == 3).filter(AwardFinancial.submission_id == self.interfaces.jobDb.getSubmissionId(jobId)).first()
```

Comment:
Just to be clear, this is the row number from the test file, not an autoincremented primary key.  Most of the rows in the test files are identical for many values.  We could give these rows specific values in some other field to query for, but that seems like it might be equally fragile.